help/destinations/catalog/data-partners/merkury-enterprise-identity.md

```diff
@@ -0,0 +1,168 @@
+UI](https://experienceleague.adobe.com/en/docs/experience-platform/destinations/ui/alerts).
+
+When you are finished providing details for your destination connection, select **Next**.
+
+## Activate audiences to this destination
+
+>[!IMPORTANT]
+>- To activate data, you need the View Destinations, Activate Destinations, View Profiles, and View Segments access control permissions. Read the access control overview or contact your product administrator to obtain the required permissions.
+>- To export identities, you need the View Identity Graph access control permission.
+
+Read [Activate audience data to batch profile export destinations](https://experienceleague.adobe.com/en/docs/experience-platform/destinations/ui/activate/activate-batch-profile-destinations) for instructions on activating audiences to this destination.
+
+## Mapping suggestions
+
+The correct processing of files on the Merkury side requires name and address elements. While not all elements are required, providing as much as possible will aid in successful matching.
+
+Mapping suggestions are provided in the table below listing attributes on your destination side that are used by Merkury processing that customers can map profile attributes to. Treat these elements as suggestions as not all elements are required, and the source values will depend on the needs of the account.
+
+|Target Field|Source Description|
+|---|---|
+|id|Identity field to be used to map merkury data to Experience Platform through the Merkury Enterprise Identity Resolution Source connector|
+|Input_First_Name|The `person.name.firstName` value in Experience Platform.|
+|Input_Last_Name|The `person.name.lastName` value in Experience Platform.|
+|Input_Address_Line_1|The `mailingAddress.street` value in Experience Platform.|
+|Input_City|The `mailingAddress.city` value in Experience Platform.|
+|Input_State_Province_Code|The `mailingAddress.state` value in Experience Platform. Use if the state is in the two character code form.|
+|Input_State_Province_Name|The `mailingAddress.state` value in Experience Platform. Use if the state is the full state name|
+|Input_Postal_Code|The `mailingAddress.postalCode` value in Experience Platform.|
+|Input_Email_Address|The value you want map as the profiles email address.|
+|Input_Phone|The value you want to map as the profiles phone number.|
+
+{style="table-layout:auto"}
+
+## Validate data export
+
+To verify if data has been exported successfully, check your Amazon S3 Storage bucket, and make sure that the exported files contain the expected profile populations.
+
+## Next steps
+
+By following this tutorial, you have successfully created a dataflow to export profile data from Experience Platform to your Merkury managed S3 location. Next, you need to contact your Merkury representative with the name of the account, file names, and the bucket path so that processing can set up.
+
+## Data usage and governance
```

Comment:
minor, but can we reverse this section and the Next steps section further above, so the article ends on the Next steps please?

Reply:
Addressed